### ISSUE TITLE
<div> force Linebreaks

### ISSUE THREAD

**Issue by michvllni** [Aymkdn/html-to-pdfmake#46]
First of all: great work, it helped me so much already.

Now to the issue:
When using `<div>` instead of `<p>` tags, html-to-pdfmake forces a new line after every span.

Example with `<div>`:
```html
<div>
	<span>hello</span><span>&nbsp;</span><span>world</span>
</div>
```
The result of html-to-pdfmake with this example:
```json
[
  {
    "stack": [
      {
        "text": "hello",
        "style": [
          "html-span"
        ]
      },
      {
        "text": " ",
        "style": [
          "html-span"
        ]
      },
      {
        "text": "world",
        "style": [
          "html-span"
        ]
      }
    ],
    "style": [
      "html-div"
    ]
  }
]
```
Same example with `<p>`:
```html
<p>
<span>hello</span><span>&nbsp;</span><span>world</span>
</p>
```
And the result:
```json
[
  {
    "text": [
      {
        "text": "hello",
        "style": [
          "html-span",
          "html-p"
        ]
      },
      {
        "text": " ",
        "style": [
          "html-span",
          "html-p"
        ]
      },
      {
        "text": "world",
        "style": [
          "html-span",
          "html-p"
        ]
      }
    ],
    "margin": [
      0,
      5,
      0,
      10
    ],
    "style": [
      "html-p"
    ]
  }
]
```
The reason for this is that using a substructure in a div always creates a stack in the generated doc definition.

**Comment by Aymkdn** [comment]
Hi,

I'll have a look later today or by the end of the week.

**Comment by Aymkdn** [comment]
I've just published v1.2.5 that should fix this issue.

Please, try and let me know!

**Comment by michvllni** [comment]
Works perfectly, thank you!
